paperspace/cli/auth.py

```diff
@@ -40,7 +41,7 @@ def logout():
 
 
 @cli.command("apiKey", help="Save your api key")
-@click.argument("api_key", required=False, callback=common.prompt_for_secret("API Key: "))
+@click.argument("api_key", required=False, callback=common.prompt_for_secret("Enter your API Key: "))
```

Comment:
will this succeed if no string is provided?

Reply:
done

Reply:
👍

Reply:
ah yes, i meant actually, could we have it fail with `API Key cannot be empty.` if the trimmed input is empty? that way we offer some minimal amount of protection/validation

Reply:
fixed